examples/share/README.md

```diff
@@ -0,0 +1,52 @@
+Tests for Lando Share Command
+=============================
+
+This example provides tests for the `lando share` command.
+
+Getting Started
+---------------
+
+This example is here for functional testing only.
+
+Helpful Commands
```

Comment:
@pirog @dustinleblanc for some reason when the parser creates the `share.spec.js` file it is getting the tests to run from this `Helpful Commands` section and not from the `Testing` section.

Reply:
parser might need some tweaks

Reply:
hmmm @serundeputy we've got other tests that have a "helpful commands" section so id look at those and compare. here is an example
https://github.com/lando/lando/tree/master/examples/python